vita3k/codec/include/codec/state.h

```diff
@@ -173,24 +176,21 @@ struct PCMDecoderState : public DecoderState {
 struct AacDecoderState : public DecoderState {
     std::shared_ptr<uint8_t> io_buffer;
     std::shared_ptr<AVIOContext> io_context;
```

Comment:
Not sure, but this two fields are not used

Reply:
ahh yes lol